dependencies.props

```diff
@@ -117,12 +113,12 @@
       <ElementName>ProjectNTfsExpectedPrerelease</ElementName>
       <BuildInfoName>ProjectNTfs</BuildInfoName>
     </XmlUpdateStep>
-    <XmlUpdateStep Condition="'$(DisableProjectNTfsAutoUpgrade)' != 'true'" Include="ProjectNTfs-TestILC">
+    <XmlUpdateStep Condition="'$(DisableProjectNTfsAutoUpgrade)' != 'true'" Include="ProjectNTfsTestILC">
```

Comment:
Doesn't these 2 update steps need to have a unique id? Right now it looks like you are only going to get the second one as they both are named ProjectNTfsTestILC and msbuild will simply include again with updated metadata (IIRC).